src/main/scala/faces/apps/RandomFaces.scala

```diff
@@ -99,24 +99,38 @@ object RandomFaces extends App {
         ))
 
         val imageData =
```

Comment:
I would prefer to integrate the occlusion as done for the expressions.
Perhaps use a seperate renderer as for other modalities and see occlusion as a modality?